courses/machine_learning/deepdive2/structured/solutions/4a_sample_babyweight.ipynb

```diff
@@ -0,0 +1,2808 @@
+{
```

Comment:
<p>Do we need both "In this notebook" and "In this lab"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/671/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>One is a general sense, the next is more detailed with the learning objectives. Once the entire first pass of labs is done for ASL 2.0, then we can do a second pass to get all of the styles more uniform so we can worry about this kind of stuff then.</p>

Reply:
<p>In some of the lab's Mike reviewed I decided to get rid of this and break from the original template. Did it here too now.</p>